ironic/drivers/modules/drac/common.py

```diff
@@ -115,3 +115,8 @@ def get_drac_client(node):
                                     driver_info['drac_protocol'])
 
     return client
+
+
+def get_reboot_options():
```

Comment:
It looks like this change is due to me resyncing my fork with Ironic.  Can you undo it?

Reply:
Done

Reply:
The change is still showing up here.  Did you forget to push the update?

Reply:
Rachit, can you please undo this change?

Reply:
Sorry i missed it..removed in latest commit